src/index.html

```diff
@@ -16,17 +16,32 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <form
+      class="search-container search-container_big"
+      data-qa="big"
+    >
+      <label class="search__label search__label_big">
+        <input
+          class="search-bar search-bar_big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
+    <form
+      class="search-container search-container_small"
+      data-qa="small"
+    >
+      <label class="search__label search__label_small">
+        <input
+          class="search-bar search-bar_small"
+          type="text"
+          data-qa="hover"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
```

Comment:
Remember about correct indentation between parent and child elements. The `form` elements and their children should have a 2-space offset from their parent `body` tag.